lib/registration.pm

```diff
@@ -79,12 +80,10 @@ sub fill_in_registration_data {
                 next;
             }
             elsif (match_has_tag('registration-online-repos')) {
-                if (!get_var('QAM_MINIMAL')) {
-                    wait_screen_change { send_key 'alt-y' };    # want updates
-                }
-                else {
-                    wait_screen_change { send_key $cmd{next} };    # minimal dont want updates
-                }
+                # don't want updates, as we don't test it or rely on it in any tests, if is executed during installation
+                # Proxy SCC replaces all update repo urls and we end up having invalid update repos, and we
+                # also do not want to have official update repos, which will lead to inconsistent SUT.
+                wait_screen_change { send_key $cmd{next} };
```

Comment:
@coolo @mimi1vx do you see a problem with changing the answer to "do you want to have updates enabled" after SCC registration from "yes" to "no" here?

Reply:
unfortunately, has fallout on our workflow.

Reply:
@mimi1vx do you have a condition in mind when we should press yes? As we want to have no as default now.

Reply:
Good thing you ask - yes, this will be a problem. See https://bugzilla.suse.com/show_bug.cgi?id=1072608 :)

Reply:
as 1058099 is fixed, can we revert to the state that worked for SP3?

Reply:
I'll create a ticket for that, as we need to verify which repos are then retruned by proxy-scc. As otherwise zypper refresh will fail due to repos we cannot access

Reply:
Will be addressed in [poo#29592](https://progress.opensuse.org/issues/29592).